integration-tests/test_unregister.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+
+
+def test_unregister(insights_client):
+    """Test insights-client --unregister after registering it
+
+    If the unregistration is successful, `unregister` method returns True
+    """
+    insights_client.register()
+    assert insights_client.is_registered
+
+    registration_status = insights_client.run("--unregister")
```

Comment:
this variable is not used -- simply do not assign the result of `run()`

Reply:
I did not understand ? It is used in assert statement.

Reply:
<facepalm>

ignore this, sorry